.pre-commit-hooks.yaml

```diff
@@ -84,3 +84,11 @@
   types: [shell]
   exclude_types: [csh, perl, python, ruby, tcsh, zsh]
   args: ['-l', '-i', '2', '-ci']
+
+- id: detect-secrets
+  name: Detect secrets
+  description: Detects high entropy strings that are likely to be passwords.
+  language: script
```

Comment:
It can use the `python` language and we install it as a dependency in our `pyproject.toml` file.

Reply:
Made it use python and added it to pyproject.toml